### DISCUSSION TITLE
How to show the file name of script that was loaded into the script editor

### DISCUSSION THREAD

**Discussion by TheXman-00** [pawelsalawa/sqlitestudio#3902] [Q&A] [answered]
I just downloaded the latest version of SQLiteStudio to check out its functionality.  Overall, I think it's very nice!  :)  

One thing that I haven't been able to find in the Wiki/FAQ/Manual or by playing around in the app, is how to determine the file name of the SQL script that is loaded into any given script editor window.  Is there something I can hover over to show me the underlying file name?  The window titles of the script editors just say "SQL editor 1", "SQL editor 2", and so on.  I'm looking for something like "C:\SQL Scripts\DB1\script1.sql".  

Thanks!

**Comment by pawelsalawa** [comment, marked as answer]
There is no file name, because when you load SQL file into the editor, it does not remain "linked" to that file. I mean that by loading file you just load its contents into the editor window and the file is forgotten. If you press  ctrl+s you will be asked where do you want to save the contents - it's because SQLiteStudio does not remember file from which it was loaded.

It could be added in future - in which case I'd suggest to raise an issue with this proposal.

**Reply by TheXman-00** [reply]
Thank you for confirming my observation.  **Until one exits the app**, the file name _**is**_ "linked" and if you hit CTRL+S it will save the file without asking for a file name.  This can be easily verified by seeing the status message saying the file name was saved.  That is what prompted me to get a definitive answer to my question.  I'm sure you can understand why it seems like a bug or omission in functionality when it knows the file name until you exit and relaunch the app.  For the sake of consistency, I would think that it either remembers the file name or it doesn't.

In any case, barring a few quirks and omissions, I think SqliteStudio is a very useful, feature-rich application, that does what it does quite well.  And the fact that it can be run as a portable app, given all that it can do, makes it even better,

Thanks and Happy New Year!
